financial-templates-lib/logger/SlackTransport.js

```diff
@@ -26,120 +26,123 @@ const SlackHook = require("winston-slack-webhook-transport");
 const { createEtherscanLinkMarkdown } = require("../../common/FormattingUtils");
 
 function slackFormatter(info) {
-  if (!("level" in info) || !("at" in info) || !("message" in info)) {
-    console.error("WINSTON INCORRECTLY CONFIGURED IN MESSAGE", info);
-    return {
+  try {
```

Comment:
The change here is to put a `try-catch` around the block. All code below did not change; the delta is due to the additional indent level.